data/sql/updates/pending_db_world/Azuregos

```diff
@@ -0,0 +1,2 @@
+
+UPDATE creature_template SET mechanic_immune_mask&~(128) WHERE (entry = 6109);
```

Comment:
```suggestion
UPDATE `creature_template` SET `mechanic_immune_mask`=`mechanic_immune_mask`&~(128) WHERE (entry = 6109);
```

Reply:
Also, you need to add `.sql` to the filename.

You can create .sql files automatically as well by using executing the `create_sql.sh` file in your `pending_db_world` folder